templates/core/index.html

```diff
@@ -6,1074 +6,382 @@
-          ) | safe
-        }}
-      </div>
+<section class="p-strip--light">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h2 class="u-sv3">Get to market fast with our strong ecosystem</h2>
+      <p>
+        <a class="p-button" href="/certified">Certified hardware</a>
+        <a class="p-button--positive" href="/core/contat-us">Contact us</a>
+      </p>
     </div>
-    <div class="col-8 col-start-large-5">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <div class=" lazyloaded" data-noscript="">
-            {{ image (
-            url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
-            alt="",
-            width="32",
-            height="32",
-            hi_def=True,
-            loading="auto"
-            ) | safe
-            }}
-          </div>
-          <p class="p-heading--4">Whitepaper</p>
+    <div class="col-6">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/67f81bfe-intel-new-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/8fdc3ebf-raspberry-pi-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/582bee22-avantech-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/75502e5c-NXP-Logo.svg" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <script>
+            let partners_logo = ["https://assets.ubuntu.com/v1/6079d02b-amd-logo.png","https://assets.ubuntu.com/v1/cd765ccc-nvidia-logo.png", "https://assets.ubuntu.com/v1/3cf238fd-Qualcomm-Logo.svg", "https://assets.ubuntu.com/v1/0176f106-rigado-logo.png","https://assets.ubuntu.com/v1/e9dabb1f-adlink+logo.png","https://assets.ubuntu.com/v1/185a14f4-Avnet.png", "https://assets.ubuntu.com/v1/97e29c13-Dell.svg","https://assets.ubuntu.com/v1/156ddbf8-rexroth_logo.svg","https://assets.ubuntu.com/v1/255a3171-abb-logo.svg","https://assets.ubuntu.com/v1/2234031b-Siemens-logo.svg.png","https://assets.ubuntu.com/v1/3ba27c02-osram-logo.svg","https://assets.ubuntu.com/v1/5bf6fd2f-svg+%281%29.svg","https://assets.ubuntu.com/v1/f6a59799-partner-logo-arm.svg","https://assets.ubuntu.com/v1/e4ac6657-xilinx-inc-logo-vector.svg","https://assets.ubuntu.com/v1/0ebb345c-MediaTek_logo.svg"]
+            const logo_section_items = document.querySelector(".p-logo-section__items")
+            let randomIndexArray = []
+            for (i = 1; i <= 5; i+=1) {
+              randomNum = Math.floor(Math.random() * partners_logo.length )
+              var logo = partners_logo[randomNum]
```

Comment:
`const` and `let` are being used in this script, so I think this `var` could be a `const` for consistency.